8.x/getting-started/contributing.md

```diff
@@ -385,6 +385,159 @@ ___
+int main(){
+    std::cout<< "Hello!";
+    return 0;
+}
+```
+
+Git makes the revision number 1 with a new random string, such as 67fe45:
+
+**OUTPUT**
+
+```bash
+#git log
+
+commit 67fe45 #(remote-name/branch-name)
+Author: #remote-account
+Date: #date-time
+
+Do some changes.
+
+#1 -> +1 -2
+```
+
+`-2` means that 2 lines are deleted: `using namespace std;` and `cout<< "Hello!";`.
+
+`+1` means that 1 new line is added: `std::cout<< "Hello!";`.
+
+In order to revert the last revision, run the following command:
+
+```bash
+git revert 67fe45
+```
+
+Instead of deleting the revision 67fe45, Git generates a new random string, such as 879012, and assigns it to a new revision number, namely 2:
+
+**OUTPUT**
+
+```bash
+#git log
+
+commit 879012 #(remote-name/branch-name)
+Author: #remote-account
+Date: #date-time
+
+Revert the last commit.
+
+#2 -> +2 -1
+```
+
+`-1` means that 1 line is deleted: `std::cout<< "Hello!";`.
+
+`+2` means that 2 lines are added: `using namespace std;` and `cout<< "Hello!";`.
+
+So the result is as follows:
+
+```bash
+#include<iostream>
+using namespace std;
+int main(){
+    cout<< "Hello!";
+    return 0;
+}
+```
+
+### Remote
+
+Git provides the capability to replicate your local repository, including the main and side branches, onto another computer accessible via the internet. This enables others to access and apply your code to their own systems. Remote servers such as `GitHub` or `GitLab`, among other Git-based platforms, may be utilized for this purpose. Typically, two remote repositories are employed: `REMOTE(Origin)` and `REMOTE(Backup)`. Therefore, it is essential to select the intended remote repository to retrieve the most recent modifications in your local system.
+
+If you are contributing to the `Larammerce-docs` project, you have already forked and cloned the REMOTE(Origin) repository from `https://github.com/larammerce/larammerce-docs`. So the larammerce-docs project on your local system pulls the data from `https://github.com/[your-account-name]/larammerce-docs` as REMOTE(Origin), while on the teammate's local system pulls from `https://github.com/[teammate-account-name]/larammerce-docs` as REMOTE(Origin).
+
+For example when you are on `branch_1` in your local system and run `git push --set-upstream origin branch_1`, the branch_1 is pushed from your local system to your REMOTE(Origin) repository, i.e. `https://github.com/[your-account-name]/larammerce-docs`.
+
+Suppose your teammate creates a new branch, named `sample-branch`, in his local system and pushes it to his REMOTE(Origin) repository and requests to merge it into the master branch of the Larammerce-docs project. In order to have your teammate's sample-branch on your local system, create a new remote, for example `REMOTE(Teammate-Origin)`, which pulls the data from `https://github.com/[teammate-account-name]/larammerce-docs` into your local repository. Run the following command on the master branch of your local repository:
```

Comment:
change `his` to` their`.

Reply:
@reyhane-rangamiz : Let's consult with the mentor.